javascript/scripts/build.mjs

```diff
@@ -93,8 +93,9 @@ function buildWasm(outputDir) {
    * "src/wasm_bindgen_output/bundler/"
    *
    * @param {string} target
+   * @param {string[]=} vitePatchFiles
    */
-  function runWasmBindgen(target) {
+  function runWasmBindgen(target, vitePatchFiles) {
```

Comment:
Rather than passing a parameter here I think we should just do the vite patching if `target == "web"`. My reasoning is that the only place this is a problem is in the `web` target because the init method generated by wasm-bindgen for the `web` target tries to guess where the wasm file is and do a dynamic import for it whereas all the other targets just hard code the location. Thus I think it's clearer that there's something about the web target we are interested in.